components/CircularLoader/CircularLoader.jsx

```diff
@@ -87,10 +96,18 @@ const CircularLoader = props => {
     },
   } = props;
 
+  const { cx, cy, r, strokeWidth } = circleProps;
+
   return (
     <Wrapper size={size} gutter={gutter} role="progressbar">
       <Content color={color} viewBox="22 22 44 44">
-        <Circle color={color} />
+        <Circle
+          STROKE-WIDTH={strokeWidth}
```

Comment:
```suggestion
          stroke-width={strokeWidth}
```

Reply:
This prop is following the material-ui, i have to check if it will not break IE.

Reply:
Ok, the ideia is just to move the property name to lower case. Is it a problem?